actionview/lib/action_view/helpers/text_helper.rb

```diff
@@ -103,7 +103,8 @@ def truncate(text, options = {}, &block)
       # Highlights one or more +phrases+ everywhere in +text+ by inserting it into
       # a <tt>:highlighter</tt> string. The highlighter can be specialized by passing <tt>:highlighter</tt>
       # as a single-quoted string with <tt>\1</tt> where the phrase is to be inserted (defaults to
-      # '<mark>\1</mark>') or passing a block that receives each matched term.
+      # '<mark>\1</mark>') or passing a block that receives each matched term. If option
+      # <tt>:sanitize</tt> is +false+, method does not sanitize +text+.
```

Comment:
I'd change this to say: "By default the text will be sanitized. something something security risk you can turn it off."